src/app/components/ScrollablePromo/index.jsx

```diff
@@ -26,13 +44,32 @@ const PromoWrapper = styled.div`
   }
 `;
 
-const ScrollablePromo = ({ blocks, blockGroupIndex }) => {
-  const { dir } = useContext(ServiceContext);
+const LabelComponent = styled.strong`
+  display: block;
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)}
+  margin-bottom: ${GEL_SPACING_TRPL};
+  color: ${C_SHADOW};
+`;
 
-  const eventTrackingData = {
-    componentName: `edoj${blockGroupIndex}`,
-    format: 'CHD=edoj',
-  };
+const getEventTrackingData = (optimizely, blockGroupIndex) => ({
+  componentName: `edoj${blockGroupIndex}`,
+  format: 'CHD=edoj',
+  ...(optimizely && { optimizely }),
+});
+
+const HINDI_EXPERIMENT_ID = '003_hindi_experiment_feature';
+
+const ScrollablePromo = ({ blocks, blockGroupIndex, isRecommendationType }) => {
+  const { script, service, dir, translations } = useContext(ServiceContext);
+  const { optimizely } = useContext(OptimizelyContext);
+
+  const promoVariation = useOptimizelyVariation(HINDI_EXPERIMENT_ID);
```

Comment:
We should be able to use the `featureId` value from `import OPTIMIZELY_CONFIG from '#lib/config/optimizely';` here.

Reply:
Did not know about that, I will make this change tomorrow morning